doc/api/cli.rst

```diff
@@ -26,6 +26,17 @@ Assume yellow tasks are outdated and green tasks are up-to-date.
 Executed tasks are shown in blue and skipped tasks are shown in white in
 diagrams below.
 
+Entry points
```

Comment:
can you follow the format from the docs? e.g., a section similar to the one in the `Build pipeline` section with a sample command and a short description.